fabfile.py

```diff
@@ -168,6 +168,33 @@ def run_migrations():
         warn("Refusing to run migrations in staging environment")
 
 
+@task
+def build_measures(environment=None, measures=None):
+    env.app = environments[environment]
+    env.environment = environment
+    env.path = "/webapps/%s" % env.app
+
+    with cd(env.path):
+        with prefix('source .venv/bin/activate'):
+            run("cd openprescribing/ && "
+                "python manage.py import_measures --measure {}".format(
+                    measures))
```

Comment:
If we're going to allow this to be called via ebmbot, it'd be nice to be able to rebuild all measures.

Reply:
I wasn't going to allow it to be called via ebmbot, on the grounds that you shouldn't need to rebuild any measures that haven't changed....?

Reply:
I was thinking of the case when `import_measures` ran incorrectly... but this hasn't happened for a long time and in any case we could run this from the console.  So scratch this suggestion.